Better issue template (#104)
leikoilja/ha-google-home: .github/ISSUE_TEMPLATE/issue.md

<!-- Before you open a new issue, search through the existing issues to see if others have had the same problem.

DO NOT DELETE ANYTHING FROM THIS TEMPLATE.

Issues not containing the minimum requirements will be closed:

- Issues without a description (using the header is not good enough) will be closed.

> - Issues without a Google version wil be closed

Reviewer: What is "Google version"?
```suggestion
- Issues without a Google version will be closed
```

> - Issues without a Google version wil be closed

Reviewer: I don't like this threatening language. How about something more kind?

Author: I just reused what was already there. Will come up with a better phrasing 😄

- Issues without any log will be closed
- Issues without debug logging will be closed.

-->

## Describe the bug

<!-- A clear and concise description of what the bug is. -->

## Version of the integration

<!-- If you are not using the newest version, download and try that before opening an issue
If you are unsure about the version check the const.py file.
-->

```text
Add version here.
```

> ## Glocaltokens Version (Underlaying package)

Reviewer: ```suggestion
## Glocaltokens Version (Underlying package)
```

<!-- Before you open a new issue, search through the existing issues to see if others have had the same problem.

DO NOT DELETE ANYTHING FROM THIS TEMPLATE.

Issues not containing the minimum requirements will be closed:

> There will be a better change of resolving this issue if the following requirements is met:

Reviewer: ```suggestion
To help us to resolve this issue please include the following information:
```

> - A good description that describes in details what the issue is.
> - Version of the integration.
> - Version of glocaltokens package.
> - Logs. The more the better.
> - Debug logs.

Reviewer: ```suggestion
- Describe the issue in detail.
- Provide steps to reproduce this issue.
- Add version of the `google_home` integration.
- Add version of `glocaltokens` package.
- Attach logs. The more the better. Enable debug logs for the integration.
```

-->

## Describe the bug

<!-- A clear and concise description of what the bug is. -->

> ## Version of the integration

Reviewer 2: Can we please mention GitHub releases (with a link) instead of 'const' file for versions?

Author: I am not sure i understand. Do you just want a link to [relaeses](https://github.com/leikoilja/ha-google-home/releases)?

Reviewer 2: Yes yes, sorry, github doesn't allow to select the lines that are not added in this PR (Lines 27-29).

I was just proposing we say:

> If you are not using the newest version, download and try that before opening an issue
> If you are unsure about the version check the [github releases page](https://github.com/leikoilja/ha-google-home/releases).

Author: Will push a commit later 😄

<!-- If you are not using the newest version, download and try that before opening an issue
If you are unsure about the version check the const.py file.
-->

```text
Add version here.
```

## Glocaltokens Version (Underlying package)

> <!-- Current installed version of the underlying package that Google Home integration rely on.

Reviewer: ```suggestion
<!-- Current installed version of the underlying package that Google Home integration relies on.
```

> <!-- Current installed version of the underlying package that Google Home integration relies on.

Reviewer 2: Currently*

> Please run `pip show glocaltokens` in the commandline and paste the information below.

Reviewer: ```suggestion
Please run `pip show glocaltokens` in the command line and paste the information below.
```

-->

```text
Add pip information here.
```

## Logs

```text
Add your logs here.
```

## Debug log

<!-- To enable debug logs check this https://www.home-assistant.io/components/logger/ -->

```text

> Add your debug logs here.

Reviewer 2: For the debug comment, shall we just put a yaml command there as helper?
